2/contrib/openshift/base-plugins.txt

```diff
@@ -7,8 +7,9 @@ openshift-client:1.0.12
 # 1.6.0 has the support we added for allowing the use of pod yaml for pod template
 # config from the config panel (there was a flavor for this already with the k8s plugin's
 # pipeline DSL, and 1.6.2 has the support for opting in the transfer of proxy related
-# env vars from the master to the client
-kubernetes:1.6.2
+# env vars from the master to the client, and 1.7.1 fixes
+# https://issues.jenkins-ci.org/browse/SECURITY-883
```

Comment:
I just updated this comment in the most basic way possible. Let me know if you'd like me to change it.